internal/errs/request.go

```diff
@@ -0,0 +1,45 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package errs
```

Comment:
I worry about naming this package `errs` since that is often a default name for packages that report multiple errors grouped by `multierr` and cause name conflicts.

Reply:
Naming being one of the hardest problems in our area! I renamed this to `colerrs`, as in `collector errors`. Not 100% happy with the name, but I'll gladly take suggestions :-)